_docs/kustomize/akash-provider/run.sh

```diff
@@ -1,7 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
 set -e
 
+pricing_script_path="/boot/pricing.sh"
+pricing_script_size=$(stat -c%s ${pricing_script_path?})
+
+if [ "$pricing_script_size" -gt 0 ]; then
```

Comment:
I think this might be too aggressive. If I'm understanding this right, if the shell script is present, then other configuration flags are ignored?

I think it will cause confusion if we include the script in the default kustomize, and then don't provide a way to use other configuration methods even when the script is present.

Reply:
OK, then what is suggested here as an alternative?

Reply:
I'd propose removing these `run.sh` changes.(sorry for the slack confusion)

Then providers could use the normal means of kustomize -> configmap settings and overrides to configure the provider, without the need to also know about this behavior here. 

If you want to use the shell script, it's configured in the same place as the other bidding behavior:
```
pricing_script_path="/boot/pricing.sh"
bid_price_strategy="shellScript"
```

Seem reasonable? That way the script can exist in the configuration and container, but not have a hard change in behavior based on existance. It will make tuning the provider's configuration easier, and I think less prone to mistakes.